source/nodejs/adaptivecards-site/themes/adaptivecards/source/js/script.js

```diff
@@ -313,6 +313,9 @@ $(function () {
 				"wrap": true
 			},
 			"spacing": 8
+		},
+		"textBlock": {
+			"headingLevel": 2
```

Comment:
TextBlock has style property and one of the style is heading.
How is `headingLevel` used?

Reply:
If the style is heading, we use `headingLevel` to set the `aria-level` attribute.

https://github.com/microsoft/AdaptiveCards/blob/8cf2a9e380b527d486007f826e83c960767a3a56/source/nodejs/adaptivecards/src/card-elements.ts#L1151-L1159

The only thing I wasn't sure about here is what we should set `headingLevel` as, so I used `2` to match the other hostConfig files.

Reply:
@paulcam206 Do you have any info on what the `headingLevel` should be for examples in the schema explorer?